k8s/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: opensergo-control-plane
+  namespace: opensergo-system
+spec:
```

Comment:
maybe give a default type `LoadBalancer`is better, user can change to `ClusterIP`or`NoderPort` by themself

Reply:
done